pkg/report/trivy.go

```diff
@@ -37,7 +37,7 @@ func (t *TrivyParser) Parse(file string) (*unversioned.UpdateManifest, error) {
 	result := trivyTypes.Result{}
 	for i := range report.Results {
 		r := &report.Results[i]
-		if r.Class == "os-pkgs" {
```

Comment:
@testwill thanks! can you update this in the test code too? https://github.com/project-copacetic/copacetic/blob/main/pkg/report/trivy_test.go#L40